src/GoodsList.tsx

```diff
@@ -1,14 +1,22 @@
+import classNames from 'classnames';
```

Comment:
one of the reasons why the red color doesn't work could be the absence of `classnames` in `package.json` dependencies. It adds automatically when you run `npm install classnames`